pkg/storage/mto_category.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
 )
 
+// MTOCategoryGetByIDsLoader returns all categories by the provided
+func MTOCategoryGetByIDsLoader(np sqlutils.NamedPreparer, _ *zap.Logger, ids []uuid.UUID) ([]*models.MTOCategory, error) {
```

Comment:
Same feedback here as in the milestone file -- should this be singular?

```suggestion
func MTOCategoryGetByIDLoader(np sqlutils.NamedPreparer, _ *zap.Logger, ids []uuid.UUID) ([]*models.MTOCategory, error) {
```